testcase/unittest/test_tx_message.py

```diff
@@ -0,0 +1,149 @@
+import os
+import random
+import threading
+import time
+
+import pytest
+
+from loopchain.baseservice import TxMessagesQueue, TxItem
+from loopchain.baseservice.tx_message import TxMessages
+from loopchain.blockchain import ExternalAddress
+from loopchain.blockchain.transactions import TransactionVersioner, TransactionBuilder
+from loopchain.crypto.signature import Signer
+
+
+@pytest.fixture
+def tx_item():
+    def _tx_item(tx_versioner):
+        test_signer = Signer.from_prikey(os.urandom(32))
+        tx_builder = TransactionBuilder.new("0x3", "", tx_versioner)
+        tx_builder.signer = test_signer
+        tx_builder.to_address = ExternalAddress.new()
+        tx_builder.step_limit = random.randint(0, 10000)
+        tx_builder.value = random.randint(0, 10000)
+        tx_builder.nid = 2
+        tx = tx_builder.build()
+        return tx
+    return _tx_item
+
+
+@pytest.fixture
+def tx_items(request, tx_item):
+    tx_versioner = TransactionVersioner()
+    tx_params = []
+
+    tx_count = request.param
+    print(f"tx_count : {request.param}")
+
+    for i in range(tx_count):
+        tx = tx_item(tx_versioner)
+        # print(f"tx : {tx.raw_data}")
+        tx_params.append((tx, tx_versioner))
+
+    return tx_params
+
+
+class TestTxMessages:
+    tx_messages = None
+    tx_count = 50
+    tx_size = 0
+
+    @pytest.mark.parametrize("tx_items", [tx_count], indirect=True)
+    def test_tx_messages(self, tx_items):
+        # given
+        self.tx_messages = TxMessages()
+        for tx_param in tx_items:
+            tx_item = TxItem.create_tx_item(tx_param, 'icon_dex')
+            self.tx_messages.append(tx_item)
+            self.tx_size += len(tx_item)
```

Comment:
Any reason to define `tx_size` as class variable? 
It looks only being used inside this test.

Reply:
ok. I will fix it.